Formula/libvncserver.rb

```diff
@@ -4,6 +4,7 @@ class Libvncserver < Formula
   url "https://github.com/LibVNC/libvncserver/archive/LibVNCServer-0.9.14.tar.gz"
   sha256 "83104e4f7e28b02f8bf6b010d69b626fae591f887e949816305daebae527c9a5"
   license "GPL-2.0-or-later"
```

Comment:
I feel like upstream should also specify the openssl linkage exception. What do you think?